src/frontend/js/components/ReferralActivityIndicator/index.tsx

```diff
@@ -193,7 +209,14 @@ export const ReferralActivityIndicator = ({
           referralurgencylevelhistory={activity.item_content_object}
         />
       );
+      break;
 
+    case ReferralActivityVerb.CLOSED:
+      messageLine2 = (
+        <ActivityCloseReferral
+          referralclosehistory={activity.item_content_object}
```

Comment:
```suggestion
          referralclosehistory={activity}
```

Right now the message is not displayed